db-migration/src/lib.rs

```diff
@@ -71,17 +83,82 @@ impl Migrations {
             .any(|m| m.expensive())
     }
 
-    /// TODO(doc): @quake
-    pub fn migrate(&self, mut db: RocksDB) -> Result<RocksDB, Error> {
-        let db_version = db
+    fn is_non_empty_rdb(&self, db: &ReadOnlyDB) -> bool {
+        let zero: Uint64 = 0u64.pack();
+        if let Ok(v) = db.get_pinned(COLUMN_INDEX, zero.as_slice()) {
```

Comment:
suggest to use `COLUMN_META`, we can remove the `ckb-types` dependencies in this crate.
```
if let Ok(v) = db.get_pinned(COLUMN_META, META_TIP_HEADER_KEY) {
```

Reply:
force push https://github.com/nervosnetwork/ckb/compare/8c99896a74ee0fcc5e4f56856084ef5eac569d48..1f417c0afe9e11b7047f7f13007ffd4b8fadda86